TeamCode/src/main/java/org/firstinspires/ftc/teamcode/hardware/VisionCamera.java

```diff
@@ -0,0 +1,52 @@
+package org.firstinspires.ftc.teamcode.hardware;
+
+import android.util.Size;
+
+import androidx.annotation.Nullable;
+
+import com.qualcomm.robotcore.hardware.HardwareMap;
+
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+import org.firstinspires.ftc.robotcore.external.hardware.camera.WebcamName;
+import org.firstinspires.ftc.teamcode.utility.Api;
+import org.firstinspires.ftc.teamcode.utility.CubeSide;
+import org.firstinspires.ftc.teamcode.utility.FieldSide;
+import org.firstinspires.ftc.vision.VisionPortal;
+
+public class VisionCamera {
```

Comment:
You need to add a comment header block describing what this class is doing.

Reply:
done